Gemfile

```diff
@@ -1,3 +1,52 @@
-eval(File.read(File.dirname(__FILE__) + '/common_spree_dependencies.rb'))
+source 'https://rubygems.org'
 
-gemspec
+gemspec require: false
+
+platforms :ruby do
+  gem 'mysql2'
```

Comment:
It seems unnecessary for me to require all database connectors at once. 
Shouldn't we only load those we actually use via `ENV['DB']`?

Reply:
I've added `require: false` here too